lib/components/Button.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class MyButton extends StatefulWidget {
+  final String text;
+  final VoidCallback onPress;
+  int keys;
+
+  MyButton(
+      {Key key,
+      this.text,
+      this.keys,
+      this.onPress})
+      : super(key: key);
+  
+  @override
+  _MyButtonState createState() => new _MyButtonState();
+}
+
+class _MyButtonState extends State<MyButton>{
+  String _displayText;
+
+  initState() {
+    super.initState();
+    print("_MyButtonState.initState: ${widget.text}");
+    _displayText = widget.text;
+
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    widget.keys++;
```

Comment:
You should not be changing the data that is passed into the widget since it can change anytime